tests/test_url_utils.py

```
@@ -0,0 +1,87 @@
+
+    def test_download_attribute(self):
+        link = MagicMock()
+        link.has_attr.return_value = True
+        is_file, ext = is_file_link("https://example.com/page", link)
+        assert is_file is True
+        assert ext == "download"
+
+    def test_no_download_attribute(self):
+        link = MagicMock()
+        link.has_attr.return_value = False
+        is_file, ext = is_file_link("https://example.com/about", link)
```

Ruff (0.15.7)

[warning] Unpacked variable `ext` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)